docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -91,48 +77,47 @@ Make sure to also review our [app development best practices](/docs/integrations
+When the merchant clicks **Install** on your single-click app, it's essential that the app successfully handle the OAuth flow before moving on to the next step of establishing a connection with the [Tax Provider API](/docs/rest-contracts/tax). The [Apps Guide](/docs/integrations/apps/guide/auth) provides an overview of the OAuth flow.
 
-When handling the OAuth flow, tax providers must ensure their app is requesting read and write permissions on the **Information and Settings** API scopes. Additionally, the tax provider should store the **client_id** and **access_token** that are received. Both are required later when [establishing a connection with the Tax Provider API](#establishing-a-connection).
+When handling the OAuth flow, you must ensure your app is requesting read and write permissions on the **Information and Settings** API scopes. Additionally, your should store the **client_id** and **access_token** that are received. Both are required later when [establishing a connection with the Tax Provider API](#establishing-a-connection).
 
 ## Establishing a connection
 
-A tax provider is ready to establish a connection with the Tax Provider API when all of the following are true:
-
-* Tax provider has shared their tax provider details with BigCommerce through email, and we've replied confirming the tax provider configuration is now ready on all supplied merchant and/or partner test stores (see [Sharing provider details with BigCommerce](#sharing-provider-details-with-bigcommerce)).
-* BigCommerce has shared the `provider_id` of the tax provider configuration with the tax provider through email (see [Sharing provider details with BigCommerce](#sharing-provider-details-with-bigcommerce)).
-* Tax provider has built a draft app that successfully handles the OAuth flow during app installation (see [Building the app](#building-the-app)).
-* During the OAuth flow the app requested the merchant to authorize read and write permissions on the Information and Settings API scopes (see [Installing the app](#installing-the-app)).
-* During the OAuth flow the `client_id` and `access_token` was received and stored by the tax provider (see [Installing the app](#installing-the-app)).
+You will be ready to establish a connection with the Tax Provider API when all of the following are true:
 
-For context, the [Tax Provider API Connection endpoints](/docs/rest-contracts/tax-app-connection) provide an added layer of security for tax providers. They set basic authentication credentials for the tax provider and these basic credentials are used to authenticate each API request to the tax provider from the associated store.
+* You have shared your tax provider details with BigCommerce through email, and we've replied confirming the tax provider configuration is now ready on all supplied merchant and/or partner test stores (see [Sharing provider details with BigCommerce](#sharing-provider-details-with-bigcommerce)).
```

Comment:
```suggestion
* You have [shared your tax provider details](#sharing-provider-details-with-bigcommerce) with BigCommerce through email, and we've replied confirming the tax provider configuration is now ready on all supplied merchant and/or partner test stores.
```

Reply:
Great, much prefer to avoid (see XXX) and have this inline 💯